CoreLibrary/src/CoreLib/CORERobot.h

```diff
@@ -16,15 +18,23 @@ class CORESubsystem;
 
 class CORERobot {
 	std::vector<CORESubsystem*> subsystems;
+
+
 public:
-//	JoystickCache joystick;
+
+	std::map<motors,CANSpeedController*> motorMap;

```

Comment:
shouldn't be a map—enum types are fixed-size, so an array is probably the correct data-structure choice, no?